packages/playground/src/portal/stats.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div>
+    <v-container>
+      <v-row>
+        <v-col v-if="Istats.length !== 0" class="d-flex flex-wrap justify-center">
+          <v-col v-for="item of Istats" :key="item.title" cols="6" md="2" sm="4" xs="12" class="px-2">
+            <StatisticsCard :item="item" />
```

Comment:
```suggestion
          <v-col v-for="item of Istats" :key="item.title" xl="2" lg="3" md="4" sm="6" cols="12" class="px-2">
            <StatisticsCard :item="item" />
```

As vuetify is mobile first approach so there is no `xs` attribute it's refered to `cols`